introduction.md

```diff
@@ -1,3 +1,7 @@
 My name is Inigo Montoya. You killed my father. Prepare to die.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?